src/components/ui/Pagination/Pagination.tsx

```diff
@@ -58,18 +58,34 @@ const Pagination = ({
           </>
         ) : (
           <>
-            {selectedNumber > 3 && (
+            {selectedNumber > 2 && selectedNumber < 4 && (
               <Link
                 className={`${styles.link}`}
                 href={`${pathPrefix}${urlID}`}
               >
                 {1}
               </Link>
             )}
-            {selectedNumber > 4 && <a className={`${styles.link} ${styles.not}`}>...</a>}
+            {selectedNumber > 3 && (
+              <>
+                <Link
+                  className={`${styles.link}`}
+                  href={`${pathPrefix}${urlID}`}
```

Comment:
Isn't it will possible to completely remove the `pathPrefix` and instead just use a relative `href` like `./strona/*`?

Reply:
It will be better from the point of view of maintenance and a fewer props